app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+def copy_file(command: str):
+    cmdList = command.split()
```

Comment:
use snake_case for naming variables
[PEP](https://peps.python.org/pep-0008/#function-and-variable-names)

Reply:
habit to call in camel Case ;)